test/parallel/test-runner-cli.js

```diff
@@ -185,3 +185,20 @@ const testFixtures = fixtures.path('test-runner');
   assert.match(stdout, /# tests 1/);
   assert.match(stdout, /# pass 1/);
 }
+
+{
+  // Use test with --loader and --require.
+  // This case is common since vscode uses --require to load the debugger.
+  const args = ['--no-warnings',
+                '--loader', join(testFixtures, 'esm', 'loader.mjs'),
+                '--test', join(testFixtures, 'index.test.js')];
```

Comment:
The argument passed to `--loader` needs to be a file URL. This code _might_ work, but more likely I’d expect it would have issues in Windows. See the loader tests that use `spawnPromisified`, they generally use `fixtures.fileURL`: https://github.com/nodejs/node/blob/e8cfa95a17cbb752af4c6f4ddc99f154021465db/test/es-module/test-loaders-workers-spawned.mjs#L43

Reply:
```suggestion
                '--experimental-loader', 'data:text/javascript,',
                '--test', fixtures.path('index.test.js')];
```
or
```suggestion
                '--experimental-loader', 'data:text/javascript,export function resolve(a,b,c){return c(a,b)}export function load(a,b,c){return c(a,b)}',
                '--test', fixtures.path('index.test.js')];
```